src/pages/create.tsx

```diff
@@ -163,9 +193,7 @@ const CreateProgramPage = () => {
                 validateProgramIdentifier(programIdentifier)
               ) {
                 setError("");
-                setLoading(true);
                 callCreateProgram();
-                setStage((prev) => prev + 1);
```

Comment:
Are we handling stuff like if you receive an error, undo the creation of the program?